src/components/AnonymousChat/child/ChatReplyPreview.tsx

```diff
@@ -82,7 +100,8 @@ const styles = StyleSheet.create({
     flex: 1,
     paddingVertical: dimen.normalizeDimen(4),
     paddingHorizontal: dimen.normalizeDimen(8),
-    borderRadius: 8
+    borderRadius: 8,
+    overflow: 'hidden'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `overflow: 'hidden'` property is added to the style. This is likely to ensure that the contents of the container (like images or text) do not spill out of the container's boundaries, especially since a borderRadius is applied.
```diff
+     borderRadius: 8,
+     overflow: 'hidden'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->